packages/common-integration-tests/src/list.ts

```diff
@@ -107,6 +107,7 @@ export function runListTests(
           IntegrationTestCacheName,
           listName
         );
+        expect(respFetch).toBeInstanceOf(CacheListFetch.Hit);
```

Comment:
this is a major improvement but does this `expect` API give us a way to define a message that will get logged if the expectation fails?  Would be really valuable to log `respFetch.toString()` here if it fails

Reply:
the error message will look something like

```
expected: CacheListFetch.Hit
received: CacheListFetch.Miss
```

Reply:
I'll +1 this, it's a big improvement, but if these keep failing I'd like to look into adding `jest-expect-message`: https://stackoverflow.com/a/53765479